tests/login.e2e.ts

```diff
@@ -4,10 +4,15 @@ import { EyesDriver } from './eyes/eyes';
 
 const loginPage = new LoginPage();
 const eye = new EyesDriver();
+const password = 'Aa123456';
+const principleUser = 'principle';
+const principleName = 'MSW PRINCIPLE';
+const teacherUser = 'teacher';
 
-const assertLoginFailure = async (t) => {
+const assertOnLoginPage = async (t) => {
   const location = await t.eval(() => window.location);
```

Comment:
please use `ClientFunction` intead
https://github.com/DevExpress/testcafe/blob/master/docs/articles/documentation/test-api/obtaining-data-from-the-client/examples-of-using-client-functions.md

Reply:
implemented.